### PR TITLE
Insert emitters and detectors at once

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

File: neurophotonics/design.py [hunk 1]
```diff
@@ -131,3 +133,4 @@ def make(self, key):
                 ),
                 skip_duplicates=True,
             )
+        self.Emitter.insert(emitter_entry, skip_duplicates=True)
```
Comment (dimitri-yatsenko) [inline neurophotonics/design.py:136]:
```suggestion
        self.Emitter.insert(emitter_entry)
```

**Comment by tdincer** [comment]
@dimitri-yatsenko The suggested changes are committed.

File: neurophotonics/design.py [hunk 2]
```diff
@@ -131,3 +133,4 @@ def make(self, key):
                 ),
                 skip_duplicates=True,
```
Comment (dimitri-yatsenko) [inline neurophotonics/design.py:134]:
```suggestion
                )
```

**Comment by tdincer** [comment]
`rotate_normal` in the last commit is to rotate the emission beam normal but not the emitter or detector itself.

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**